src/Resources/config/datagrid/brand.yml

```diff
@@ -2,90 +2,101 @@ datagrid:
-                    label: Label
-                    data_name: rd.label
-                created:
-                    type:             date
-                    label:            Created
-                    data_name:        rd.created
-                    filter_by_having: true
-                updated:
-                    type:             date
-                    label:            Updated
-                    data_name:        rd.udapted
-                    filter_by_having: true
-        sorters:
-            columns:
-                code:
-                    data_name: rd.code
-                label:
-                    data_name: rd.label
-                created:
-                    data_name: rd.created
-                updated:
-                    data_name: rd.udapted
-            default:
-                code: '%oro_datagrid.extension.orm_sorter.class%::DIRECTION_ASC'
+                label: Logo Url
         properties:
             id: ~
             edit_link:
                 type: url
-                route: pim_customentity_edit
+                route: pim_customentity_rest_get
                 acl_resource: pim_cnet_enrich_brand_edit
                 params:
                     - id
                     - customEntityName
             delete_link:
                 type: url
-                route: pim_customentity_delete
+                route: pim_customentity_rest_delete
                 acl_resource: pim_cnet_enrich_brand_delete
                 params:
                     - id
                     - customEntityName
         actions:
             edit:
                 type:      navigate
-                label:     brand.edit
                 acl_resource: pim_cnet_enrich_brand_edit
+                label:     brand.edit
                 icon:      edit
                 link:      edit_link
                 rowAction: true
             delete:
                 type:  delete
-                label: brand.delete
                 acl_resource: pim_cnet_enrich_brand_delete
+                label: brand.delete
                 icon:  trash
                 link:  delete_link
+        filters:
+            columns:
+                code:
+                    type:      string
+                    label:     Code
+                    data_name: rd.code
+                label:
+                    type:  search
+                    label: Label
+                    data_name: rd.label
+                logo_url:
+                    type: string
+                    label: Logo URL
+                    data_name: rd.logoUrl
+
+        sorters:
+            columns:
+                code:
+                    data_name: rd.code
+                label:
+                    data_name: rd.label
+                logo_url:
+                    data_name: rd.logoUrl
+            default:
+                code: '%oro_datagrid.extension.orm_sorter.class%::DIRECTION_ASC'
+
+        mass_actions:
```

Comment:
Why did you add these two mass actions on the datagrid?

Reply:
Because it was easy to do :), but I can remove them if you prefer